libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -311,11 +313,12 @@ - (void)enqueueRequest:(SFRestRequest *)request requestDelegate:(id<SFRestReques
                 [strongSelf notifyDelegateOfSuccess:requestDelegate request:request data:dataForDelegate rawResponse:response];
             } else {
                 if (shouldRetry && statusCode == 401) {
-
-                    // 401 indicates refresh is required.
-                    [strongSelf replayRequest:request response:response requestDelegate:requestDelegate];
+                    // Do not refresh token if biometric authentiction lock is enabled
```

Comment:
does `notifyDelegateOfFailure:` need to be called if the lock is enabled? or is it not the same as a failure?

Reply:
Good call, I will combine the if surrounding the replay request with the one above.